scripts/check-runtime-versions.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Required minimum versions.
+REQUIRED_GO_VERSION=$(grep 'version:' .promu.yml | awk '{print $2}')
+REQUIRED_NODE_VERSION=$(cat web/ui/.nvmrc | tr -d '\r' | sed 's/v//')
+
+# Function to compare versions (checks if version A >= version B).
+compare_versions() {
```

Comment:
I think sth with `if ! echo -e "${min_version}\n${version}" | sort --version-sort --check=silent; then...` should be sufficient.

Reply:
Thanks for the suggestion. I made the checks one liners.